Mapping/ClassMetadataInterface.php

```diff
@@ -77,7 +77,7 @@ public function getReflectionClass();
     /**
      * Gets the attribute identifier of the class.
      *
-     * @return AttributeMetadataInterface
+     * @return AttributeMetadataInterface|null
```

Comment:
I've just updated the doc to reflect the reality, but I'll change in deep the identifier system.